spec.md

```diff
@@ -2999,18 +2999,29 @@ or infinity).
 
 If x is a `bool`, the result is 0 for `False` or 1 for `True`.
 
-If `x` is a string, it is interpreted as a sequence of digits in the specified
-base, decimal by default. If `base` is zero, `x` is interpreted like an integer
-literal, the base being inferred from an optional base marker such as `0b`,
-`0o`, or `0x` preceding the first digit. These markers may also be used if
-`base` is the corresponding base. Irrespective of base, the string may start
-with an optional `+` or `-` sign indicating the sign of the result.
+If x is a string, it is interpreted as a sequence of digits in the
```

Comment:
"sequence of digits in the specified base" -- specify that the base can range from 2 to 36 inclusive, with a...z representing digits 11 through 35.

Reply:
Fixed.